custom/code_types.inc.php

```diff
@@ -47,11 +47,18 @@
  * @author    Jerry Padgett <sjpadgett@gmail.com>
  * @copyright Copyright (c) 2006-2010 Rod Roark <rod@sunsetsystems.com>
  * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2021 Robert Down <robertdown@live.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+use OpenEMR\Events\Codes\ExternalCodesCreatedEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+require_once dirname(__FILE__) . "/../interface/globals.php";
```

Comment:
requiring this line in a library generally means something is wrong in the script that is using this library. Guessing that whatever is requiring this line in your module is potentially causing your issue.